### PR TITLE
Cargo: change tikv-server master version to 6.4.0-alpha

### LINKED ISSUE #13562: when `tikv:6.3.0` checkout to `tikv:master` in k8s, k8s pod cannot start and `crashloopbackoff`
### What version of TiKV are you using?

v6.3.0
### Steps to reproduce

`tikv:6.3.0` checkout to `tikv:master` in k8s env
### What did you expect?
checkout successfully
### What did happened?
1. k8s pod `tikv-7` cannot start and `crashloopbackoff`
065523f1-5e1b-4d06-8050-6998211e6cd3
14293804-9b8d-4c03-9b9e-a16441468965
2. and logs in loki show that pd client assumed tikv version is v6.2.0-alpha
img_v2_4a6e0c40-9c83-4887-bc28-dc4e59d3816g

### PR DESCRIPTION
**Pull request by BornChanger**
Signed-off-by: BornChanger <dawn_catcher@126.com>

### What is changed and how it works?

Issue Number: Close #13562 

What's Changed:

```
Change tikv-server version to 6.4.0-alpha
```

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
verify tikv-server --version output

Side effects

N/A

### Release note 

```release-note
Sync up tikv-server version to 6.4.0-alpha
```

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tikv"
-version = "6.2.0-alpha"
+version = "6.3.0-alpha"
```
Comment (BusyJay) [inline Cargo.toml:3]: Should be 6.4.0 actually. If 6.3.0 is released, master should be 6.4.0-alpha.

Comment (BornChanger) [reply Cargo.toml:3]: let's keep the pace of TiDB and PD at present to let go the k8s installing at present. And then make another change later.

Comment (BusyJay) [reply Cargo.toml:3]: TiDB and PD should also be changed to 6.4.0-alpha.

Comment (BornChanger) [reply Cargo.toml]: > TiDB and PD should also be changed to 6.4.0-alpha.

OK.

File: Cargo.lock [hunk 2]
```diff
@@ -6144,7 +6144,7 @@ dependencies = [
 
 [[package]]
 name = "tikv"
-version = "6.2.0-alpha"
+version = "6.3.0-alpha"
```
Comment (BusyJay) [inline Cargo.lock:6147]: Lock file is not updated.

**Review by BusyJay — APPROVED** [review]

**Comment by BornChanger** [comment]
I noticed it duplicated with https://github.com/tikv/tikv/pull/13499.

**Pull request closed without merging**